pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -720,3 +726,14 @@ def embed_plot_in_desc(plot):
         raise ValueError(msg)
 
     return desc
+
+
+def plot_from_config(config, factory_map=DEFAULT_FACTORIES):
+    """ Build plot factory capable of building a plot described by config.
+    """
+    plot_type = config.plot_type
+    plot_factory_klass = factory_map[plot_type]
+    factory = plot_factory_klass(**config.to_dict())
+    desc = factory.generate_plot()
+    plot = desc["plot"]
+    return plot, factory, desc
```

Comment:
Good question. I am trying to reuse this function inside the normal plot generation process, rather than just in the unit test. If that's the case, we will need all 3 of these objects.